src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewModel.swift

```diff
@@ -95,6 +95,13 @@ struct AntigenTestProfileViewModel {
 		store.antigenTestProfile = nil
 	}
 
+	mutating func refreshProfile() {
+		guard let antigenTestProfile = store.antigenTestProfile else {
+			fatalError("We can't refresh without a valid antigenTestProfile stored")
```

Comment:
Hm sure we want to crash here?

Reply:
Its the same in the init. I used it here too, to avoid an optional antigenTestProfile